index.js

```diff
@@ -1,11 +1,44 @@
 'use strict';
 const fs = require('fs');
+const os = require('os');
 const path = require('path');
 const childProcess = require('child_process');
 const runApplescript = require('run-applescript');
 const pify = require('pify');
 const rimraf = require('rimraf');
-const xdgTrashdir = require('xdg-trashdir');
```

Comment:
I think it would be better to make [`xdg-trashdir`](https://github.com/kevva/xdg-trashdir) support returning all the trash dirs instead of having all the logic here.

```js
xdgTrashdir.all();
```